### PR TITLE
BAH-3263 |Deepthi M/Deepti R|Added GHA workflow to build apk

### PR DESCRIPTION
**Pull request by deepthi-mantena**

### CODE REVIEW

File: .github/workflows/build_apk.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Sample GitHub Actions Workflow
```
Comment (mohan-13) [inline .github/workflows/build_apk.yml]: Pls update the name of he workflow

File: .github/workflows/build_apk.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+on:
+  push:
+    branches:
+      - master
+
+  workflow_dispatch:
+
+jobs:
+  avni-apk:
```
Comment (mohan-13) [inline .github/workflows/build_apk.yml]: You can also better name the job. `build-apk` something like that

File: .github/workflows/build_apk.yml (hunk 1 above)
Comment (deepthi-mantena) [reply .github/workflows/build_apk.yml]: Updated Mohan

File: .github/workflows/build_apk.yml (hunk 2 above)
Comment (deepthi-mantena) [reply .github/workflows/build_apk.yml]: yes modified

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**